IntegrationDevelopment/Help/IntegrationDevelopmentGuide.postman_collection.json

```diff
@@ -229,6 +229,115 @@
 						}
 					]
 				},
+				{
+					"name": "New journal transaction with changed business date and branch",
+					"request": {
+						"method": "PUT",
+						"header": [
+							{
+								"key": "Accept",
+								"value": "application/json",
+								"type": "text"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							},
+							{
+								"key": "PX-CbApiBusinessDate",
+								"value": "2018/01/01",
+								"type": "text"
+							},
+							{
+								"key": "PX-CbApiBranch",
+								"value": "SweetLife Store",
+								"type": "text"
+							}
+						],
```

Comment:
@PetrKorchagin , please add a description of the request. For an example, see other requests in this folder of the collection.

Reply:
Done.